erpnext/projects/doctype/task/task.py

```diff
@@ -101,6 +107,39 @@ def update_project(self):
 		if self.project and not self.flags.from_project:
 			frappe.get_doc("Project", self.project).update_project()
 
+	def update_project_tasks(self):
+		tasks = frappe.db.sql("""SELECT name
```

Comment:
fix indentation https://github.com/frappe/erpnext/wiki/Coding-Standards#use-simple-indenting

Reply:
Also use ORM

Reply:
I think that this is solved now, can you confirm?